v1/pipeline/steps/cd/sync-step-info.yaml

```diff
@@ -49,6 +51,11 @@ allOf:
       oneOf:
         - type: boolean
         - type: string
+  oneOf: 
+  - required: 
+    - applicationsList
+  - required:
+    - applicationRegex
```

Comment:
do we need the same logic that we have below here?

Reply:
Yes fixing